search-index/index_data.py

```diff
@@ -41,38 +41,50 @@ def get_document_generator(
     # Create name_and_id field to group by and sort on in Elasticsearch aggregation.
     # TODO: this needs to be a combination of document name and document ID when we remove
     # the concept of actions.
-    main_dataset["action_name_and_id"] = (
-        main_dataset["action_name"] + " " + main_dataset["document_id"].astype(str)
+    main_dataset["document_name_and_id"] = (
+        main_dataset["document_name"] + " " + main_dataset["document_id"].astype(str)
     )
 
-    main_dataset["action_date"] = main_dataset["action_date"].apply(
+    main_dataset["document_date"] = main_dataset["document_date"].apply(
         lambda i: i.strftime("%d/%m/%Y")
     )
-    main_dataset["action_description"] = main_dataset["action_description"].str.strip()
+    main_dataset["document_description"] = main_dataset[
+        "document_description"
+    ].str.strip()
 
     # --------------------------------------------------------------------------------------------
 
     metadata_columns = [
         "document_id",
         "document_language_id",
         "document_name",
-        "action_id",
-        "action_date",
-        "action_name",
-        "action_description",
-        "action_name_and_id",
-        "action_country_code",
-        "action_geography_english_shortname",
-        "action_source_name",
-        "action_type_name",
+        "document_date",
```

Comment:
Everything prepended by "action" has been removed and replaced with document ids. This list should now contain everything expected according to 480.